.gitignore

```diff
@@ -3,4 +3,9 @@ keybaseca.config
 nohup.out
 env.sh
 __pycache__
-generated-env
\ No newline at end of file
+generated-env
```

Comment:
i think this is old now, no? looks like it's only in the tests directory.

Reply:
Yup, that directory no longer exists or is needed. Removed from the gitignore (if you still have it, just blow it away).